src/mixxx.cpp

```diff
@@ -507,13 +507,16 @@ void MixxxMainWindow::initialize(QApplication* pApp, const CmdlineArgs& args) {
 
     // Load skin to a QWidget that we set as the central widget. Assignment
     // intentional in next line.
-    if (!(m_pWidgetParent = m_pSkinLoader->loadConfiguredSkin(this, m_pKeyboard,
-                                                              m_pPlayerManager,
-                                                              m_pControllerManager,
-                                                              m_pLibrary,
-                                                              m_pVCManager,
-                                                              m_pEffectsManager,
-                                                              m_pRecordingManager))) {
+    m_pWidgetParent = m_pSkinLoader->loadConfiguredSkin(this,
+            &m_skinCreatedControls,
+            m_pKeyboard,
+            m_pPlayerManager,
+            m_pControllerManager,
+            m_pLibrary,
+            m_pVCManager,
+            m_pEffectsManager,
+            m_pRecordingManager);
+    if (!m_pWidgetParent) {
         reportCriticalErrorAndQuit(
                 "default skin cannot be loaded see <b>mixxx</b> trace for more information.");
```

Comment:
side note: this message is quite useless. even if you read 'trace' as 'mixxx log file', IIRC I did never find the cause in there but by recapping the steps I did earlier (mistake in git, wrong link to skins directory, ...) which is tedious.

Reply:
Do you have a better idea for the message?

Reply:
didn't look into it yet, just noticed. 
it can fail for various reasons and maybe we can print that.

Reply:
Ok, but we can do that in a separate PR.